### PR TITLE
Cache composer dependencies

### PR DESCRIPTION
**Pull request by driesvints**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <a href="https://packagist.org/packages/laravel/framework"><img src="https://img.shields.io/packagist/l/laravel/framework" alt="License"></a>
 </p>
 
-## About Laravel
+## About Laravel 2
```
Comment (antonkomarev) [inline README.md:10]: Mistake?

Comment (driesvints) [reply README.md:10]: Intentional to see the effect of the cache.

**Comment by driesvints** [comment]
Doesn't seems to be giving the wanted results.

**Comment by GrahamCampbell** [comment]
Typically caching will be slower.

**Pull request closed without merging**